tools/aws-ip.sh

```diff
@@ -1,3 +1,8 @@
 ipv4=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
 
-echo '\n\nAPI_URL=http://'$ipv4':3000' >> ../telescope/.env
+cp config/env.development config/env.remote
+
+sed -i "s/for Development/for Remote Development/g" config/env.remote
+sed -i "s/localhost/$ipv4/g" config/env.remote
```

Comment:
I wonder if we should look for `http://localhost` vs. `localhost` on its own?  I'm trying to think if we'll ever regret changing `localhost` to an ip automatically, and breaking something in the future.  Maybe it's fine, I'm just pondering...